crates/bevy_reflect/src/impls/std.rs

```diff
@@ -1019,6 +1100,27 @@ impl FromReflect for Cow<'static, str> {
     }
 }
 
+impl Typed for &'static Path {
+    fn type_info() -> &'static TypeInfo {
+        static CELL: NonGenericTypeInfoCell = NonGenericTypeInfoCell::new();
+        CELL.get_or_set(|| TypeInfo::Value(ValueInfo::new::<Self>()))
+    }
+}
+
+impl GetTypeRegistration for &'static Path {
+    fn get_type_registration() -> TypeRegistration {
+        let mut registration = TypeRegistration::of::<Self>();
+        registration.insert::<ReflectFromPtr>(FromType::<Self>::from_type());
```

Comment:
Are there other registrations we should include here? I think we could do `ReflectSerialize` but I'm not sure `ReflectDeserialize` is possible for a reference (?) so maybe we shouldn't include either? 🤔

Reply:
Hm, I don't think there is anything else to register apart from those. So this is probably fine.

Reply:
It might make more sense to support serialization for `Cow<'static, Path>` instead.